roles/cockpit_run_verify/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+# vim: set ft=ansible:
+#
+- name: Set cockpit container name for RHELAH
+  set_fact:
+    cockpit_cname: "registry.access.redhat.com/rhel7/cockpit-ws"
+  when: ansible_distribution == "RedHat"
+
+- name: Set cockpit container name for Fedora/CentOS
+  set_fact:
+    cockpit_cname: "cockpit/ws"
+  when: ('CentOS' in ansible_distribution) or ansible_distribution == "Fedora"
+
+- name: Run cockpit container
+  command: atomic run {{ cockpit_cname }}
```

Comment:
I'd separate `docker pull {{ cockpit_cname }}` from `atomic run`.  And have a separate timeout (at least a minute for download, but just ~10 seconds for run?